flycheck.el

```diff
@@ -3685,7 +3690,16 @@ Intended for use with `next-error-function'."
   (let ((pos (flycheck-next-error-pos n reset)))
     (if pos
         (goto-char pos)
-      (user-error "No more Flycheck errors"))))
+      (let ((result (cl-remove-if
```

Comment:
We try not to use cl functions in Flycheck.  Please replace `cl-remove-if` with the equivalent from seq.el, or from dash.el if seq.el has no such function.